triggers/service/src/main/scala/com/digitalasset/daml/lf/engine/trigger/ServiceConfig.scala

```diff
@@ -139,9 +140,14 @@ object ServiceConfig {
       .valueName(JdbcConfig.usage)
       .text("JDBC configuration parameters. If omitted the service runs without a database.")
 
+    opt[Unit]("no-secret-key")
+      .action((_, c) => c.copy(noSecretKey = true))
+      .text("Allow running without a secret key.")
```

Comment:
Some more warning here might be helpful, e.g.
```suggestion
      .text("Allow running without a secret encryption key. This means that user credentials are not secure so this should not be used if security is a concern.")
```